docs/porting/target/bootstrap.md

```diff
@@ -54,9 +54,9 @@ If you are using the below linker script, then you need to update all the define
   #define MBED_APP_SIZE  MBED_ROM_SIZE
 #endif
 
-#if !defined(MBED_BOOT_STACK_SIZE)
+#if !defined(MBED_CONF_TARGET_BOOT_STACK_SIZE)
```

Comment:
Is this really useful?
In Mbed environment, this macro has to be defined... or we have some other issue ?
Which use case do you protect ?
Thx

Reply:
The intended (and likely desired) behavior here is to prefer MBED_CONF_TARGET_BOOT_STACK_SIZE if present, and to fall back to MBED_BOOT_STACK_SIZE if not. If MBED_BOOT_STACK_SIZE is not defined, then we fall back to a hard-coded value.

Reply:
Yes, I can see this :-)
But which uses case do you see to add this fall back system ?